### ISSUE TITLE
Pie and Donut chart error while hovering on legend instead highlighting

### ISSUE THREAD

**Issue by chojnicki** [apexcharts/apexcharts.js#4710]
### Description

While hovering over pie chart legend by default it should highlight corresponding series on chart. Instead there is error repeated every cursor move. 

`Uncaught TypeError: Cannot read properties of undefined (reading 'toString')`

To avoid it for now I can disable this functionality:

```
legend: {
  onItemHover: {
    highlightDataSeries: false,
  },
},
```

I know there is no issue with my config because same issue is on official site.

### Steps to Reproduce

1. Go to any pie chart example like https://apexcharts.com/javascript-chart-demos/pie-charts/simple-pie-chart/
2. Move cursor over legent.
3. Viola!

### Expected Behavior

Pie chunk should highlight.

### Actual Behavior

Errors in console and no effect.

### Screenshots

![Screenshot from 2024-09-17 14-42-34](https://github.com/user-attachments/assets/687fba46-be88-43a1-9330-0acebfd9662c)

### Reproduction Link

https://apexcharts.com/javascript-chart-demos/pie-charts/simple-pie-chart/

**Comment by nithin-pb** [comment]
facing the same issue, downgrading to `3.52.0` is fixed this issue for me, it looks like only in the latest version

**Comment by nadar** [comment]
Yes we have the same problem :-(

**Comment by nadar** [comment]
i think this is fixed and should be released soon: https://github.com/apexcharts/apexcharts.js/issues/4683

**Comment by eviltik** [comment]
fix available in the latest version 3.54.0, thx !

**Comment by chojnicki** [comment]
Yeah, it works now so I'm closing it.
